app/templates/add_resource.html

```diff
@@ -31,7 +31,14 @@ <h1 class="page-header">{{ title }}</h1>
 	    <div class="row">
 		    <div class="large-8 medium-8 medium-centered small-8 small-centered
 		     columns large-centered">
-		      	<label>{{ form.name.label }}
+		    {% if field.flags.required %}
+		      	<label>{{ field.label(class_='required') }}
+		    {% else %}
+				<label>{{ field.label }}
+		    {% endif %}
+		    		<div class="form-field-description">
+		      			{{ field.description }}
+		      		</div>
```

Comment:
Yes, the indentation looks fine in sublime. I would say that github is just displaying it oddly.